<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { background-color: white; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="schuifdeurKnop">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hallsensor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>s1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideur1Knop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideur2Knop">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="schuifPassSlot1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiCodeSlot2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiCodeSlot1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="schuifCodeSlot1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiPassSlot1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter password:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser { background-color: white; color:black}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knopToegang">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Access</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knopGeenToegang">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No access</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiHerkenSlot1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter: identification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kaartenbakKnop">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKaartenbak">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cardbox Access</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idInput">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="naamInput">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="adresInput">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>420</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIdKaart">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>340</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Card</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelId">
    <property name="geometry">
     <rect>
      <x>754</x>
      <y>362</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNaam">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>392</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdres">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>422</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knopMaakIdKaart">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add new card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knopVerwijderIdKaart">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>480</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove card</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="schuifKaartSlot1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter ID number:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kaartToegang">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Access</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kaartGeenToegang">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No access</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="draaiKaartSlot1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white; color: black;}</string>
    </property>
    <property name="placeholderText">
     <string>Enter ID number:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
